src/Schema/Directives/Nodes/GroupDirective.php

```diff
@@ -77,14 +69,26 @@ protected function setMiddlewareDirectiveOnFields($objectType)
             return $objectType;
         }
 
-        $middlewareValues = '["'.implode('", "', $middlewareValues).'"]';
-        $middlewareDirective = PartialParser::directive("@middleware(checks: $middlewareValues)");
+        $middlewareValues = addslashes(
```

Comment:
`addslashes` is causing issues here:

```graphql
extend type Query @group(middleware: ["some", "other"] {
   someField: Type @all
}
```

incorrectly expands to

```graphql
@middleware(checks: ["some\", \"other"])
```

Having only 1 middleware in `@group`-Directive works but if there are 2 or more, it fails with:
```
{"errors":[{"debugMessage":"Function name must be a string","message":"Internal server ...
```

Reply:
Nice find, was an easy fix. I just `addslashes` before imploding the array now. Thanks!